packages/styles/forms.css

```diff
@@ -168,7 +168,11 @@ textarea.Field--has-error:focus:hover,
   display: block;
   position: relative;
   box-sizing: border-box;
-  margin-bottom: var(--space-smallest);
+  margin-bottom: var(--space-small);
+}
+
+.Field .Error {
+  min-height: unset;
```

Comment:
Error has its min-height for 24px, which makes the gap between erroring TextField or Select too large.

Reply:
@schne324 @scurker Do you think we could just get rid of the min-height from the `.Error` class? Looking at walnut and the extension, we only ever conditionally render the element that has an error on it (e.g. `{error && <div className="Error">Something went wrong</div>}`